pkg/apis/serving/fieldmask.go

```diff
@@ -710,10 +710,12 @@ func SecurityContextMask(ctx context.Context, in *corev1.SecurityContext) *corev
 	// SeccompProfile defaults to "unconstrained", but the safe values are
 	// "RuntimeDefault" or "Localhost" (with localhost path set)
 	out.SeccompProfile = in.SeccompProfile
-
+	// Allow setting Privileged to only false
+	if in.Privileged != nil && !*in.Privileged {
```

Comment:
Pls add a test case in `k8s_validation_test.go`. [Here is an example of this.](https://github.com/knative/serving/compare/main...skonto:serving:privileged_false?expand=1)